trade-finance-manager-api/src/v1/controllers/user/user.routes.js

```diff
@@ -129,22 +127,22 @@ module.exports.login = async (req, res, next) => {
 
   if (loginResult.error) {
     // pick out the specific cases we understand and could treat differently
-    if (userNotFound === loginResult.error) {
+    if (userOrPasswordIncorrect === loginResult.error) {
       return res.status(401).json({
         success: false,
-        msg: `Could not find the ${username} user`,
+        msg: 'Username or password incorrect',
       });
     }
-    if (incorrectPassword === loginResult.error) {
+    if (userIsDisabled === loginResult.error) {
       return res.status(401).json({
         success: false,
-        msg: 'You entered the wrong password',
+        msg: 'User is disabled',
       });
     }
-    if (userIsDisabled === loginResult.error) {
+    if (userIsBlocked === loginResult.error) {
       return res.status(401).json({
         success: false,
-        msg: 'User is disabled',
+        msg: 'Account is blocked',
```

Comment:
Was there a reason we used `Account` instead of `User` for this message?

Reply:
Ah no - I was going to change the other one, which is `account` to `user`!